dx/usr/bin/bluenfin-incus

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env bash
+
+# All the pieces needed to enable incus on Bluefin-dx
+# This script is meant to be run on a bluefin-dx host
+
+# if current user is root, warn and exit
+
+if [ "$EUID" -eq 0 ]
+then
+    echo "This script should not be run as root"
+    exit
+fi
+
+CURRENT_IMAGE=$(rpm-ostree status -b --json | jq -r '.deployments[0]."container-image-reference"')
+
+if grep -q "/var/ublue-os/image" <<< $CURRENT_IMAGE
+then
+    echo ""
+    echo "Before we can enable Incus on this system"
+    echo "the current system needs an update. Please run 'just update'"
+    echo "and reboot your system when the update is finished."
+    exit
+fi
+
+if grep -q "bluefin-dx" <<< $CURRENT_IMAGE
+then
+    CURRENT_STATE="dx"
+else
+    CURRENT_STATE="not-dx"
+fi
+
+# if we aren't on the dx image, we need to rebase
+# to the dx image
+
+if [ "$CURRENT_STATE" = "dx" ]
+then
+    echo "You are already on a developer image"
+else
+    echo ""
+    echo "Before we can enable Incus on this system"
+    echo "the current system needs to be on a developer (dx)"
+    echo "image. Please run 'just devmode' to rebase to a developer image"
+    echo "and reboot your system when the rebase is finished."
+    exit
+fi
```

Comment:
I don't think you need most of this logic.  This script will only be present on the `dx` images, so there's no need to check if this is a DX image or not

Reply:
I guess there's no harm in leaving it here though.

Reply:
i flip/flopped on this, thinking someone might see it in the repo, or in a github search. I can go either way. Tiebreaker @castrojo ?

Reply:
If you're thinking about if this is used outside ublue, maybe some checks to make sure the required CLI tools are installed, rather than checking the exact image?
I would expect this script to not function at all on a random Ubuntu VM since it's expecting the `ublue-os/image` file to be present, exiting if it is not.

E.G. 
```
CHECK INCUS_CLI EXISTS
IF NOT EXIST:
  OUTPUT: "Incus CLI is not installed.  Please install it.  If on Bluefin, run `just devmode on`"
ELSE:
  CONTINUE
```